File: src/minesweeper/c#/Minesweeper.Tests/MinefieldTests.cs
namespace Minesweeper.Tests;
using FluentAssertions;

public class MinefieldTest
{
    [Fact]
    public void ShouldShowEmptyFields()
    {
        new Minefield(new [] { "." }).CalculateHints()
            .Should().Equals(new [] { "0" });
    }
}

File: src/minesweeper/kotlin/kotlin/src/test/kotlin/MinefieldTests.kt
package minesweeper.tests

import minesweeper.Minefield
import org.junit.jupiter.api.Assertions.assertArrayEquals
import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class MinefieldTests{
    @Test
    fun `it should return empty string`() {
        val minefield = Minefield(arrayOf("."))

        val actual = minefield.calculateHints()
        val expected = arrayOf("0")

        assertArrayEquals(expected, actual)
    }
}